microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Build Microsoft Edge sidebar PWAs
+description: How to build your PWA for installation in the Microsoft Edge Sidebar.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+ms.date: 04/27/2023
+---
+# Build Microsoft Edge sidebar PWAs
+
+![Experimental feature](../../media/experimental-tag.png)
+
+Progressive Web Apps (PWAs) can opt-in to be pinned to the sidebar in Microsoft Edge.
+
+The sidebar in Microsoft Edge allows users to easily access popular websites and utilities alongside their browser tabs. The content in the sidebar augments the user's primary task by enabling side-by-side browsing and minimizing the need to switch contexts between browser tabs.
+
+By signaling intent to be pinned to the sidebar, your PWA gets the following benefits:
+
+* Your PWA can be promoted in the Microsoft Edge sidebar store.
+* Your PWA can detect whether it is running in the Microsoft Edge sidebar or not.
+* Your PWA defines its own style and layout to provide a user-friendly sidebar experience.
+
+
+<!-- ====================================================================== -->
+## Enable sidebar support of PWAs in Microsoft Edge
+
+Experimental support for pinning PWAs in the Microsoft Edge's sidebar starts with Microsoft Edge 112. Some parts of the feature are available starting with Microsoft Edge 114, such as the ability to detect if your PWA is running in the sidebar by using User Agent Client Hints.
```

Comment:
Experimental support for PWAs that can target the sidebar in Microsoft Edge starts with ...